.github/workflows/ci.yml

```diff
@@ -231,6 +231,44 @@ jobs:
       - name: Run yarn test262
         run: exit 0 # TODO remove test262 from required tests for CI
 
+  test-hermes:
+    name: test-hermes
+
+    # begin macro
```

Comment:
what macro?

Reply:
was going to remove these comments, but kept in line with our other 6 jobs

they're TODO's from yrs ago to eventually replace everyth between them with a reusable step/workflow, so i've opened
- https://github.com/endojs/endo/issues/2457

Reply:
Oh! These macros predate me and I still haven’t taken the trouble to figure out enough Github YAML to solve them, but I am getting close.